### ISSUE TITLE
#899384014------------- Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1376]
<strong>Source:</strong>`<span class="p-button-label ng-star-inserted" data-pc-section="label">Draft</span>`

<strong>Element Location: </strong>.p-disabled > .p-button-label[data-pc-section="label"]

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.55 (foreground color: #848484, background color: #f9f9f9, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,TTv5,TT13.c,EN-301-549,EN-9.1.4.3,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=axeAPI
